### DISCUSSION TITLE
change in siwave syz simulation behavior between v0.7.3 and v0.7.4

### DISCUSSION THREAD

**Discussion by ghost** [ansys/pyaedt#4117] [Q&A]
It seems I am not able to set up a siwave syz simulation anymore after the v0.7.4 release; I've noticed some of the examples have changed to reflect the design being imported into hfss3dlayout for simulation. Is it not possible to set up a siwave syz simulation after this release?

**Comment by Samuelopez-ansys** [comment]
It is possible, you should use PyEDB to define the setup which is much efficient:

https://examples.aedt.docs.pyansys.com/version/dev/examples/high_frequency/layout/power_integrity/power_integrity.html
